Wiki.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
This wiki aims to summarize the class diagrams for the main library packages in TracerX that is built upon KLEE repository.

**1. Overview**

In TracerX library, the klee and kleaver code is organized as follows:
- lib/Basic: Low level support for both klee and kleaver, independent of LLVM.
- lib/Support: Higher level support that is only used by klee, and using LLVM facilities mainly.
- lib/Expr: The core kleaver expression library.
- lib/Solver: The kleaver solver library.
- lib/Module: klee facilities to function together with LLVM modules, which include the shadow module/instruction structures used during execution.
- lib/Core: The core symbolic virtual machine